### PR TITLE
Generate data for hadoop

### PR DESCRIPTION
**Pull request by gregflack**
- Add queries to hive
- Add data generator for hadoop
- Update of POM file

### CODE REVIEW

File: integ-tests/hadoop/src/main/java/com/google/testdata/TestDataHelper.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    }
+
+    SqlHelper.executeQueries(connection, queryList);
+
+    LOGGER.info(
+        String.format(
+            "Generated %s new constraints:%n%s",
+            queryList.size(), Joiner.on(lineSeparator()).join(queryList)));
+
+    setEnv("SCHEMAS", dbName);
```
Comment (dmorokov) [inline integ-tests/hadoop/src/main/java/com/google/testdata/TestDataHelper.java:70]: Why do we do this?

File: integ-tests/hadoop/src/main/java/com/google/testdata/Constants.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * limitations under the License.
+ */
+package com.google.testdata;
+
+/** Stores constants common among all tests. */
+public final class Constants {
+
+  public static final String HIVE_HOST = "localhost";
+  public static final String DB_NAME = "default";
+  public static final String USERNAME_DB = "";
```
[the remark this answers is not in the document]
Comment (gregflack) [reply integ-tests/hadoop/src/main/java/com/google/testdata/Constants.java:23]: By default hive doesn't have username and password but this is required to establish connection. I can remove it and pass empty string as parameter.

File: integ-tests/hadoop/src/main/java/com/google/testdata/Constants.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.testdata;
+
+/** Stores constants common among all tests. */
+public final class Constants {
```
[the remark this answers is not in the document]
Comment (gregflack) [reply integ-tests/hadoop/src/main/java/com/google/testdata/Constants.java:19]: sure, done

File: integ-tests/hadoop/src/main/java/com/google/testdata/Constants.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+  public static final String HIVE_HOST = "localhost";
+  public static final String DB_NAME = "default";
+  public static final String USERNAME_DB = "";
+  public static final String PASSWORD_DB = "";
+  public static final String URL_DB = String.format("jdbc:hive2://%s:10000/%s", HIVE_HOST, DB_NAME);
+
+  public static final String SQL_TEST_DATA_BASE_PATH = "src/main/java/com/google/sql/";
+  public static final String SCHEMA_FILE = "tables.jsonl";
```
[the remark this answers is not in the document]
Comment (gregflack) [reply integ-tests/hadoop/src/main/java/com/google/testdata/Constants.java:28]: yes this path contains all sql files that I use

File: integ-tests/hadoop/src/main/java/com/google/testdata/TestDataHelper.java (hunk 1 above)
Comment (gregflack) [reply integ-tests/hadoop/src/main/java/com/google/testdata/TestDataHelper.java:70]: this is part of next PR that will come after configuring extraction in kokoro - I've removed it for now

File: integ-tests/hadoop/src/main/java/com/google/testdata/Constants.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (dmorokov) [reply integ-tests/hadoop/src/main/java/com/google/testdata/Constants.java]: Will be moved to different PR, fine.

File: integ-tests/hadoop/src/main/java/com/google/testdata/TestDataHelper.java (hunk 1 above)
Comment (dmorokov) [reply integ-tests/hadoop/src/main/java/com/google/testdata/TestDataHelper.java]: Please, follow up with Christian on best practices for situations like this.

File: integ-tests/hadoop/src/main/java/com/google/testdata/Constants.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (gregflack) [reply integ-tests/hadoop/src/main/java/com/google/testdata/Constants.java]: Done

File: integ-tests/hadoop/src/main/java/com/google/testdata/TestDataHelper.java (hunk 1 above)
Comment (gregflack) [reply integ-tests/hadoop/src/main/java/com/google/testdata/TestDataHelper.java]: Ok, I will clarify with him and include it in next PR

File: integ-tests/hadoop/pom.xml [hunk 5]
```diff
@@ -21,36 +21,69 @@
...
+
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-jdbc</artifactId>
+      <version>${hive.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.glassfish</groupId>
+      <artifactId>javax.el</artifactId>
```
Comment (haselbach) [inline integ-tests/hadoop/pom.xml:58]: The license of this dependency is problematic. IIUC it is GPLv2 or CDDC. GPLv2 is a no-go. CDDC requires us to make the source code available (extra work).

Can we remove the dependency?

Comment (gregflack) [reply integ-tests/hadoop/pom.xml:58]: Yes, I removed it.

**Review by haselbach — APPROVED** [review]

**Pull request merged**